locales/en/plugin__mcg-ms-console.json

```diff
@@ -2,30 +2,29 @@
   "Bucket name": "Bucket name",
   "Data source type": "Data source type",
   "Single data source": "Single data source",
-  "The bucket will read and write its data to a selected data source": "The bucket will read and write its data to a selected data source",
+  "The bucket reads and writes data to a single data source.": "The bucket reads and writes data to a single data source.",
   "Multi data source": "Multi data source",
-  "The bucket will serve reads from several selected data sources, creating a virtual namespace on top of them and will write to one or more of those as its chosen write target": "The bucket will serve reads from several selected data sources, creating a virtual namespace on top of them and will write to one or more of those as its chosen write target",
+  "The bucket reads from several selected data sources, creating a virtual namespace on top of them, and writes into one of those.": "The bucket reads from several selected data sources, creating a virtual namespace on top of them, and writes into one of those.",
   "Namespace": "Namespace",
-  "To allow access to this bucket, an ObjectBucketClaim will be created in the below specified namespace in the cluster. Select the namespace in which it will be created": "To allow access to this bucket, an ObjectBucketClaim will be created in the below specified namespace in the cluster. Select the namespace in which it will be created",
+  "Creates object bucket claim in the specified namespace.": "Creates object bucket claim in the specified namespace.",
```

Comment:
```suggestion
  "Creates ObjectBucketClaim in the specified namespace.": "Creates object bucket claim in the specified namespace.",
```

Reply:
why are we doing this? any particular reason?